packages/next/src/build/index.ts

```diff
@@ -1946,6 +1947,9 @@ export default async function build(
                 processCwd: config.experimental.turbotrace.processCwd,
                 logDetail: config.experimental.turbotrace.logDetail,
                 showAll: config.experimental.turbotrace.logAll,
+                memoryLimit:
+                  config.experimental.turbotrace.memoryLimit ??
+                  TURBO_TRACE_DEFAULT_MEMORY_LIMIT,
```

Comment:
Instead of using a constant and importing it everywhere, could you instead set a default value like we do here?

https://github.com/vercel/next.js/blob/e8fcf8b569341d639757ff86cf26667e8b1be159/packages/next/src/shared/lib/image-config.ts#L100-L114

Reply:
The default value of `experimental.turbotrace` is `undefined`: https://github.com/vercel/next.js/blob/canary/packages/next/src/server/config-shared.ts#L622